docs/public/using-devfile-odo.dev.push.path-attribute.adoc

```diff
@@ -0,0 +1,29 @@
+# Using the odo.dev.push.path related attributes
+
+`odo` uses the `odo.dev.push.path` related attribute from the devfile's run commands to push only specific files and folders to the component. The format of the attribute is `odo.dev.push.path:<local_relative_path>:<remote_relative_path>`
```

Comment:
```suggestion
`odo` uses the `odo.dev.push.path` related attribute from the devfile's run commands to push only a specific folder and its contents to the component. The format of the attribute is `odo.dev.push.path:<local_relative_path>:<remote_relative_path>`
```

Reply:
But we can mention and push individual files as well.

Reply:
It is not clear to me what can be pushed. Can we push only one file or directory+contents, or can we specify several ones?

Reply:
> Can we push only one file or directory+contents, or can we specify several ones?

Yes we can mention and push only a single file to the remote destination as well. In case of a folder, all of it contents will be pushed to the mentioned remote path. We can also mention several such paths.

Reply:
OK, I see. So your sentence is exact, but you could say in the introduction that one can mention several paths, and what is the syntax to do this

Reply:
I have added a mention about the use of several such paths and have also added it to the example.